### PR TITLE
Split up future of iota

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Moved and split up the content from `Future of IOTA` into the Shimmer docs. 
I also rewrote the NFT article to more describe native assets and how the interact with L2.
Wdyt?

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: shimmer/learn/native-assets.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - asset wrapping
+  - Layer 1
+  - Layer 2
+---
+
+![tokenized-assets](/img/learn/layer2-tokenized-assets.png)
+
+Layer 1 Assets are also known as "[Native Assets](https://github.com/iotaledger/tips/pull/38)". With the
+[IOTA tokenization framework](https://blog.iota.org/tokenization-on-the-tangle-iota-digital-assets-framework/), you can
+create NFTs and new Tokens on L1 using the new output types. Native assets can be transferred on the Tangle without any fees and between different smart contract chains.
```
Comment (lucas-tortora) [inline shimmer/learn/native-assets.md:18]:
```suggestion
Layer 1 Assets are also known as "[Native Assets](https://github.com/iotaledger/tips/pull/38)". With the
[IOTA tokenization framework](https://blog.iota.org/tokenization-on-the-tangle-iota-digital-assets-framework/), you can
create NFTs and new Tokens on L1 using the new output types. You can transfer native assets on the Tangle without any fees and between different smart contract chains.
```

File: shimmer/learn/native-assets.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Native Tokens
+
+In Shimmer you can basically create Native out of thin air as long as you fullfill the [storage deposit](/learn/welcome#storage-deposits).  
+Native tokens can have different types of metadata.
```
Comment (lucas-tortora) [inline shimmer/learn/native-assets.md:23]:
```suggestion
In Shimmer, you can create Native out of thin air as long as you fulfill the [storage deposit](/learn/welcome#storage-deposits).  
Native tokens can have different types of metadata.
```

File: shimmer/learn/native-assets.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+:::
+
+## NFTs
+
+Ethereum was the first blockchain to support NFTs with its ERC-721 standard. The difference from normal tokens is that the NFT is unique and clearly verifiable and can be used to prove ownership of any digital good.
+
+There are many use cases for NFTs. The most common use case is to link an image to it, which becomes a unique digital art which can be sold on open NFT marketplaces.
+
+Other use cases would be to use NFTs as assets in games, for event tickets, for Decentralized Finance (DeFi), for voting
+in decentralized autonomous organizations (DAOs) or for any objects in the Metaverse.
```
Comment (lucas-tortora) [inline shimmer/learn/native-assets.md:39]:
```suggestion
Ethereum was the first blockchain to support NFTs with its ERC-721 standard. The difference from regular tokens is that the NFT is unique and verifiable and can be used to prove ownership of any digital good.

There are many use cases for NFTs. The most common use case is to link an image to it, which becomes a unique digital art that can be sold on open NFT marketplaces.

Other use cases would be to use NFTs as assets in games, for event tickets, for Decentralized Finance (DeFi), for voting
in decentralized autonomous organizations (DAOs) or for any objects in the Metaverse.
```

File: shimmer/learn/role-of-token.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+title: Role of the Shimmer Token
+description: Introduction to the emerging permissionless Smart Contracts network
+keywords:
+  - smart contracts
+  - permissionless
+---
+
+With the introduction of ISC(IOTA Smart Contracts) it's important to understand the role of the base token. ISC is build on top of the Tangle, it will need the Tangle for anchored state updates, interoperability between chains, for Native Assets and general access to a congested network. There are two main usages of the base token in ISC and everyting build on top of Shimmer.
+
+## The new storage deposit
```
Comment (lucas-tortora) [inline shimmer/learn/role-of-token.md:12]:
```suggestion
## Storage Deposits
```

**Review by lucas-tortora — APPROVED** [review]
Looks good to me :) I think we'll need to review the learn section as a whole when everything is merged.

**Review by jlvandenhout — APPROVED** [review]

**Pull request merged**